pkg/orm/util.go

```diff
@@ -83,8 +85,8 @@ type MetaOpsClient struct {
 // Initialize will create all related tables in SQL backend
 // TODO: What if we change the definition of orm??
 func (c *MetaOpsClient) Initialize(ctx context.Context) error {
-	if err := c.db.AutoMigrate(&model.ProjectInfo{}, &model.ProjectOperation{}, &model.MasterMeta{},
-		&model.WorkerMeta{}, &model.ResourceMeta{}); err != nil {
```

Comment:
Can we remove `ResourceMeta` definition from `pkg/orm/model/resource.go` in this PR, ditto for several merged models.

Reply:
ok